docker/install/ubuntu_install_tensorflow.sh

```diff
@@ -21,6 +21,5 @@ set -u
 set -o pipefail
 
 pip3 install \
-    "h5py<3.0" \
```

Comment:
It would also be good to increment the version of `h5py`, otherwise installing tensorflow gives:
```
ERROR: tensorflow 2.6.2 has requirement h5py~=3.1.0, but you'll have h5py 2.10.0 which is incompatible.
```